library/Haste/Util/Url.php

```diff
@@ -33,7 +33,12 @@ public static function addQueryString($strQuery, $varUrl = null)
         $strUrl = static::prepareUrl($varUrl);
         $strQuery = trim(ampersand($strQuery, false), '&');
 
-        list($strScript, $strQueryString) = explode('?', $strUrl, 2);
+        $arrUrl = explode('?', $strUrl, 2);
+
+        // Prevent undefined array key warning
+        $arrUrl[1] = $arrUrl[1] ?? '';
+
+        list($strScript, $strQueryString) = $arrUrl;
```

Comment:
```suggestion
        list($strScript, $strQueryString) = explode('?', $strUrl, 2) + array('', '');
```

That would be shorter (and that's how the Contao Core does it).

Reply:
This is cool! Thank you!